cypress/support/config/settings.js

```diff
@@ -5250,58 +5250,58 @@ module.exports = () => ({
         environments: {
           live: {
             paths: [
-              '/persian/articles/c7eel0lmr4do', // Article
-              '/persian/afghanistan', // IDX
-              '/persian/popular/read', // Most Read
-              '/persian/magazine-49281981', // CPS PGL
-              '/persian/world-51497110', // CPS MAP
-              '/persian/media-49522521', // CPS MAP with live stream
-              '/persian/world/2016/06/160613_om_naked_dining', // TC2 MAP
-              '/persian/bbc_persian_radio/liveradio', // Live Radio
-              '/persian/bbc_dari_radio/liveradio', // Live Radio
-              '/persian/bbc_dari_radio/programmes/p0340v0s', // On Demand Radio Brand
-              '/persian/bbc_persian_radio/programmes/p0340vyx', // On Demand Radio Brand
-              '/persian/bbc_dari_radio/w3ct0bst', // On Demand Radio Episode
-              '/persian/bbc_persian_radio/w3ct0s49', // On Demand Radio Episode
+              // '/persian/articles/c7eel0lmr4do', // Article
```

Comment:
I think just setting `enabled` to false is enough - shouldn't need to comment all these out as well

Doesn't do any harm though

Reply:
You are right, it usually does. For some reason this time it kept running the tests even though I had changed it to false! Maybe it wasn't updating correctly. I got fed up and just commented them out to make it work.